ydb/docs/en/core/concepts/topology.md

```diff
@@ -23,3 +23,110 @@ The storage volume multiplier specified above only applies to the fault toleranc
+
+### Simplified hardware configurations {#reduced}
+
+If it's not possible to use the [recommended amount](#cluster-config) of hardware, you can divide servers within a single rack into two dummy fail domains. In this configuration, a failure of 1 rack means a failure of 2 domains and not a single one. In [both fault tolerance modes](#fault-tolerance), {{ ydb-short-name }} will keep running if 2 domains fail. If you use the configuration with dummy fail domains, the minimum number of racks in a cluster is 5 for `block-4-2` mode and 2 in each data center for `mirror-3-dc` mode.
+
+### Fault tolerance level {#reliability}
+
+The table below describes fault tolerance levels for different fault tolerance modes and hardware configurations of a {{ ydb-short-name }} cluster:
+
+Fault tolerance<br>mode | Fail<br>domain | Fail<br>realm | Number of<br>data centers | Number of<br>server racks | Fault tolerance<br>level
+:--- | :---: | :---: | :---: | :---: | :---
+`block-4-2` | Rack | Data center | 1 | 9 or more | Can stand a failure of 2 racks
+`block-4-2` | ½ a rack | Data center | 1 | 5 or more | Can stand a failure of 1 rack
+`block-4-2` | Server | Data center | 1 | Doesn't matter | Can stand a failure of 2 servers
+`mirror-3-dc` | Rack | Data center | 3 | 4 in each data center | Can stand a failure of a data center and 1 rack in one of the two other data centers
+`mirror-3-dc` | Server | Data center | 3 | Doesn't matter | Can stand a failure of a data center and 1 server in one of the two other data centers
+
+## Node Broker Configuration {#node-broker-config}
+
+Node Broker is a system tablet that registers dynamic nodes in the {{ ydb-short-name }} cluster.
+
+Node broker gives names to dynamic nodes when they register. By default, a node name consists of the hostname and the port on which the node is running.
+
+In a dynamic environment where hostnames often change, such as in Kubernetes, using hostname and port leads to an uncontrollable increase in the number of unique node names. This is true even for a database with a handful of dynamic nodes. Such behavior may be undesirable for a time series monitoring system as the number of metrics grows uncontrollably. To solve this problem, the system administrator can set up *stable* node names.
+
+A stable name identifies a node within the tenant. It consists of a prefix and a node's sequential number within its tenant. If a dynamic node has been shut down, after a timeout, its stable name can be taken by a new dynamic node serving the same tenant.
+
+To enable stable node names, you need to add the following to the cluster configuration:
+
+```yaml
+feature_flags:
+  enable_stable_node_names: true
+```
+
+By default, the prefix is `slot-`. To override the prefix, add the following to the cluster configuration:
+
+```yaml
+node_broker_config:
+  stable_node_name_prefix: <new prefix>
+```
```

Comment:
This looks unrelated to distributed storage and topologies. I'd keep it where it was for now.

Reply:
Sure, fixed that (that was an error, moved NB config back to config)